krux/cli.py

```diff
@@ -329,7 +333,13 @@ def add_logging_args(parser):
     group.add_argument(
         '--log-file',
         default=None,
-        help='Full-qualified path to the log file '
+        help='Full-qualified path to the log file'
```

Comment:
You need a trailing space in this string or it will run together with the default.